### PR TITLE
Fixed ADMozaikLayoutCache's retain cycle & added tvOS support

### PR DESCRIPTION
**Pull request by pawlowskialex**

### CODE REVIEW

File: ADMozaikCollectionViewLayout/Private/ADMozaikLayoutCache.swift [hunk 1]
```diff
@@ -82,6 +82,9 @@ class ADMozaikLayoutCache {
         if let size = self.cachedSizeOfItemAtIndexPathDictionary[indexPath] {
             return size
         }
-        return self.mozaikLayoutDelegate.collectionView(self.collectionView, mozaik: layout, mozaikSizeForItemAt: indexPath)
+        guard let delegate = self.mozaikLayoutDelegate else {
```
Comment (Antondomashnev) [inline ADMozaikCollectionViewLayout/Private/ADMozaikLayoutCache.swift]: I think it makes sense to check this before let above as this is a required parameter.

**Review by Antondomashnev — COMMENTED** [review]
Thanks for your contribution @pawlowskialex 🚀 
Do you use it on tvOS?

I have one comment and after that it's good to go 😄

File: ADMozaikCollectionViewLayout/Private/ADMozaikLayoutCache.swift (hunk 1 above)
Comment (pawlowskialex) [reply ADMozaikCollectionViewLayout/Private/ADMozaikLayoutCache.swift]: sure, it does, fixed now

**Comment by pawlowskialex** [comment]
Yeah, I'm using it on tvOS now, works like charm!

**Comment by Antondomashnev** [comment]
> Yeah, I'm using it on tvOS now, works like charm!

Amazing, glad to hear that 😄

**Comment by Antondomashnev** [comment]
Ah sorry, the last one I forgot and I haven't setup [Danger](danger.systems) for this yet. Could you update the changelog?

**Comment by pawlowskialex** [comment]
Sure, done.

**Comment by Antondomashnev** [comment]
@pawlowskialex will prepare a release today.

**Pull request merged**